### DISCUSSION TITLE
email-validator update

### DISCUSSION THREAD

**Discussion by rorik302** [dymmond/esmerald#156] [Potential Issue]
I've found a problem with email-validator == 1.3.1. When I try to use `EmailStr` I recieve `"AttributeError(\"'ValidatedEmail' object has no attribute 'normalized'\")"`. If I update email-validator manually to 2.0.0 the problem is gone. Because Esmerald use `openapi-schemas-pydantic` first of all need to update email-validator in it. Especially PR already exists.

**Comment by tarsil** [comment]
That is a great thing @rorik302 . I will update straightway!! Nice catch

**Comment by tarsil** [comment]
@rorik302 the new release of openapi-schemas-pydantic is now out. This week a new Esmerald release will come out and this will be included. Just install the latest OpenAPI schemas pydantic and that's it 🙂

**Comment by tarsil** [comment]
@rorik302 good news. The updates are already merged but not yet deployed and the reason being it's because I'm doing some updates to the documentation and it should be out next week maximum but like I mentioned before, for now just update the openapi-schemas-pydantic that is already in the version 3 and you should be good to go :)

**Comment by tarsil** [comment]
@rorik302 Esmerald 2.0.6 has just been released https://esmerald.dev/ and brings a shinny new landing page in case you are also interested. The issues you found with the requirements were addressed as well

**Comment by tarsil** [comment]
@rorik302 I'm assuming you now have the latest version of Esmerald and therefore I can close this discussion. Anything else, just reply 👍🏼
